app/views/provider_interface/provider_users_invitations/edit_details.html.erb

```diff
@@ -13,6 +13,11 @@
       <%= f.govuk_text_field :first_name, label: { text: 'First name', size: 'm' } %>
       <%= f.govuk_text_field :last_name, label: { text: 'Last name', size: 'm' } %>
 
+      <% if current_provider_user.providers.size == 1 %>
+        <%= f.hidden_field :single_provider, value: true %>
```

Comment:
Maybe we could set this on the wizard as soon as the user starts, rather than posting it back?

Reply:
@duncanjbrown rebased into https://github.com/DFE-Digital/apply-for-teacher-training/pull/2606/commits/03120fcb607db8c66e193d7f4bf9a6ef087226c5

Reply:
@duncanjbrown We skip the orgs page here https://github.com/DFE-Digital/apply-for-teacher-training/pull/2606/commits/239d6baba7115a59f53ac217463f1c4a90882646#diff-2397cfec288e45bd0d29bd8a1c428527R99 is this not working as expected for you? Seems to work on PR app...